site/docs/guidelines/tables-lists.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "Tables vs lists"
+navTitle: "Tables vs lists"
+summaryParagraph:
+tags: ["Data Tables", "Tabular Data", "Cards", "Tiles", "Rows", "Grids"]
+inComparingSection: true
+---
+
+- **Tables also known as:** Data Tables, Tabular Data
+- **Lists also known as:** Cards, Tiles, Rows, Grids
+
+## Need to know
+### Tables
+- Tables are for presenting information which gives the user the flexibility to sort the data to find their own patterns
+- Tables must include column headers for sorting
+
+### Lists
+- Lists are for presenting grouped information into scannable patterns. The content within a list can be stacked to create hierarchy within the data.
+- Lists are sortable by a ‘Sorted by’ Select, and do not have header labels
+- A single row of a list could make sense
+
+### General
```

Comment:
I would put this above the tables and list rules - either in the summary or immediately after the Need to know heading

Reply:
Took it out completely - put one line in the summary, the other as a standalone bullet in Need to Know